include/uio.h

```diff
@@ -21,6 +21,20 @@ typedef struct uio {
   vm_map_t *uio_vmspace; /* destination address space */
 } uio_t;
 
+/* Uses struct subtyping enabled by `-fplan9-extensions` flag described in:
+ * https://gcc.gnu.org/onlinedocs/gcc/Unnamed-Fields.html */
+typedef struct uio_single {
```

Comment:
Ok. You're right that this code does justify introduction of Plan9 extensions to C language. We'll revisit this idea at the next meeting. Hopefully I'll be able to show some better use cases for it.